### ISSUE TITLE
unknown error occurred in faster whisper

### ISSUE THREAD

**Issue by cyhycf** [chidiwilliams/buzz#849]
buzz 1.0.1 windows
gpu ram 6g
cuda 12.5
Systran/faster-whisper-large-v3 can work on other projects

Tested faster whisper by import file and found unknown errors in both large-v3 large-v2 and medium models

```
file_transcriber.run:61 ERROR -> 
Traceback (most recent call last):
  File "buzz\transcriber\file_transcriber.py", line 59, in run
  File "buzz\transcriber\whisper_file_transcriber.py", line 80, in transcribe
Exception: Unknown error
```

In addition, besides the large-v3 model, it will be downloaded from guillaumekln and Systran to Buzz and huggingface cache directory twice

**Comment by raivisdejus** [comment]
There was a defect where Buzz crashed if "Detect language" was selected. I think you may be getting this case. 
Try selecting what language to transcribe.

**Comment by Code-1123** [comment]
The same thing happened to me

**Comment by raivisdejus** [comment]
@cyhycf @Code-1123 An issue with faster whisper model was fixed. Please test latest development version https://github.com/chidiwilliams/buzz/actions/runs/10211346492
You need to log into github to see the artifacts section with installation files.

**Comment by raivisdejus** [comment]
Faster whisper does not work on CUDA enabled Windows systems. The very latest development versions will not try to use GPU / CUDA on Windows for faster whisper, but it should work with CPU. Get the latest version from https://github.com/chidiwilliams/buzz/actions?query=branch%3Amain or wait for release `1.0.2`.

Regular Whisper and Huggingface whisper should work also on Windows. See this note on enabling CUDA support on Windows https://github.com/chidiwilliams/buzz/blob/main/CONTRIBUTING.md#gpu-support

**Comment by raivisdejus** [comment]
Fixed in latest version `1.1.0`
